codes/javascript/chapter_computational_complexity/time_complexity.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * File: time_complexity.js
+ * Created Time: 2023-01-02
+ * Author: RiverTwilight (contact@rene.wang)
+ */
+
+class time_complexity {
+	/* 常数阶 */
+	constant(n) {
+		var count = 0;
+		const size = 100000;
+		for (var i = 0; i < size; i++) count++;
```

Comment:
We use "four spaces" as the indentation, not "tab", for the JS and TS code.

Reply:
So I recommend to add a prettier config in the root directory:D

Reply:
Feel free to add it~